From 92ed1e297632a718d1392c8d163beb713c00ccbf Mon Sep 17 00:00:00 2001
From: Lans Zhang <jia.zhang@windriver.com>
Date: Wed, 9 Aug 2017 16:29:08 +0800
Subject: [PATCH 5/5] fallback: always try to boot the option recorded in csv

We intend to use fallback to work around MSFT for the next bootloader
of shim. Thus, we don't mind fallback is involved for PCR measurement
at all.

Signed-off-by: Lans Zhang <jia.zhang@windriver.com>
---
 fallback.c | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/fallback.c b/fallback.c
index 2794cc1..0a645a4 100644
--- a/fallback.c
+++ b/fallback.c
@@ -1016,13 +1016,7 @@ efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *systab)
 		return rc;
 	}
 
-	rc = fallback_should_prefer_reset();
-	if (EFI_ERROR(rc)) {
-		VerbosePrint(L"tpm not present, starting the first image\n");
-		try_start_first_option(image);
-	} else {
-		VerbosePrint(L"tpm present, resetting system\n");
-	}
+	try_start_first_option(image);
 
 	Print(L"Reset System\n");
 
-- 
2.7.5

